build.gradle

```diff
@@ -251,6 +251,9 @@ allprojects {
                     force "io.netty:netty-common:${nettyVersion}"
                     force "io.netty:netty-codec-http:${nettyVersion}"
 
+                    force "org.springframework:spring-core:${springVersion}"
+                    force "org.springframework:spring-expression:5.3.22"
```

Comment:
Perhaps add a comment about why this forcing is done (and why the versions are different).  Are we forcing both because forcing spring-expression then tries to update spring-core?